.gitignore

```diff
@@ -37,3 +37,10 @@ yarn-error.log*
 
 # typescript
 *.tsbuildinfo
+
+# tauri
+/src-tauri/gen
+
+# IDE
```

Comment:
Let's not exclude IDE folders in the gitignore here. Since this is a template repository, it's likely for some users to fork and configure their own settings for their forked repository.